easybuild/easyconfigs/t/TRIQS/TRIQS-2.2.0-foss-2019a-Python-2.7.15.eb

```
@@ -0,0 +1,56 @@
+easyblock = 'CMakeMake'
+
+name = 'TRIQS'
+version = '2.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://triqs.github.io/'
+description = """
+ TRIQS (Toolbox for Research on Interacting Quantum Systems) is a
+ scientific project providing a set of C++ and Python libraries to
+ develop new tools for the study of interacting quantum systems.
+"""
+
+docurls = ['https://triqs.github.io/triqs/%(version_major_minor)s.x/reference.html']
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/TRIQS/triqs/archive/']
```

Can you change this to
```suggestion
sources = ['triqs-%(version)s.tar.gz']
source_urls = ['https://github.com/TRIQS/triqs/releases/download/%(version)s/']
```

---

This is meant to be a replacement of both lines, not just one. Somehow this wasn't picked up